util/nodeReleaseData.ts

```diff
@@ -0,0 +1,37 @@
+export const getNodeReleaseStatus = (
+  now: Date,
+  currentStart?: string,
+  ltsStart?: string,
+  maintenanceStart?: string,
+  endOfLife?: string
+) => {
+  if (endOfLife) {
+    const endOfLifeDate = new Date(endOfLife);
+    if (now > endOfLifeDate) {
```

Comment:
```suggestion
    if (now > new Date(endOfLife)) {
```

You can simplify this for each statement. No need to create a constant for each date as they're used only inside the if check